mathics/builtin/drawing/image.py

```diff
@@ -2089,7 +2089,7 @@ def get_sort_key(self, pattern_sort=False) -> list:
             # but with a `2` instead of `1` in the 5th position,
             # and adding two extra fields: the length in the 5th position,
             # and a hash in the 6th place.
-            return [1, 3, SymbolImage, tuple(), 2, len(self.pixels), hash(self)]
+            return tuple((1, 3, SymbolImage, tuple(), 2, len(self.pixels), hash(self)))
```

Comment:
Here and many other places the `tuple()` is superfluous. Given that the return type now is listed as `tuple`, in my opinion that is sufficient. `dict({1: 2})` in my view isn't any clearer than `{1: 2}`.

Reply:
OK, I removed the `tuple(...)`. I left the `dict(...)` because it was there before. In a subsequent round, I can remove them.